src/utils/BulbBotUtils.ts

```diff
@@ -670,6 +670,7 @@ export default class {
 	public diff<T>(oldObj: T, newObj: T): string[] {
 		const diff: string[] = [];
 		for (const key of Object.keys(oldObj)) {
+			if (key === null || oldObj === null || oldObj[key] === null) continue;
```

Comment:
Are the first two cases even possible? My instinct would be to just use optional chaining for the `valueOf` calls on the line below, but I can see a problem with `null` handling in `objectEquals`, so I'll open an issue for that

Reply:
oh yeah those should not be possible